ui/pages/confirmations/components/simulation-details/fiat-display.test.tsx

```diff
@@ -5,17 +5,10 @@ import { merge } from 'lodash';
 import { useFiatFormatter } from '../../../../hooks/useFiatFormatter';
 import { renderWithProvider } from '../../../../../test/lib/render-helpers';
 import mockState from '../../../../../test/data/mock-state.json';
-import { CHAIN_IDS } from '../../../../../shared/constants/network';
 import { IndividualFiatDisplay, TotalFiatDisplay } from './fiat-display';
 import { FIAT_UNAVAILABLE } from './types';
 
-const mockStateWithTestnet = merge({}, mockState, {
-  metamask: {
-    providerConfig: {
-      chainId: CHAIN_IDS.SEPOLIA,
-    },
-  },
-});
+const mockStateWithTestnet = merge({}, mockState, {});
```

Comment:
nit: `mockState` could be used directly here

Reply:
put this back to sepolia to match previous mock: https://github.com/MetaMask/metamask-extension/pull/26150/commits/12ca1d3867cf8f370d147693d867f24b87bfc86e